neuraxle/checkpoints.py

```diff
@@ -40,7 +40,7 @@ class DataCheckpointType(Enum):
     EXPECTED_OUTPUT = 'eo'
 
 
-class BaseCheckpointer(NonFittableMixin, NonTransformableMixin, BaseStep):
+class BaseCheckpointer(NonTransformableMixin, BaseStep):
```

Comment:
it seems to me that the basestep would be replaced with a transformer since youve removed the nonfittablemixin. AM I right or is this irrelevant ? it seems that this shouldn't fit and will crash if we try to fit on it.

Reply:
no NonTransformerMixin is fine. it allows me to not implement transform because BaseCheckpointer doesn't need it.